controllers/front/CartController.php

```diff
@@ -636,15 +647,15 @@ protected function areProductsAvailable()
 
         if ($product['active']) {
             return $this->trans(
-                'The item %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.',
-                ['%product%' => $product['name']],
+                'The product %product% in your cart is no longer available in this quantity. You cannot proceed with your order until the quantity is adjusted.',
```

Comment:
```suggestion
                'The product %product% in your cart is no longer available in this quantity. Please adjust the quantity to proceed with the order.',
```
I tried to modify this wording a bit to make it sound less negative.